pandas/core/frame.py

```diff
@@ -10088,6 +10088,20 @@ def map(
         0  3  4
         1  5  5
 
+        >>> def func(x,y): # x refers to the element in the DataFrame
```

Comment:
Can you just define a lambda that takes an extra argument?

Reply:
Actually my goal was to demonstrate how the map function can be used with standard functions that are not lambdas. If you think I should use a more complex example to show that, I could do so.

Reply:
Can you just provide a built in function like `round`?

Reply:
Thank you for the advice! I did just that, you can have a look.